### ISSUE TITLE
[a11y-defect-2]: Benefit Letters form flow must use the VADS spec for progress bars with headings

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#96393]
### Point of contact

Trevor Pierce

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

The Benefits Letters form flow has an older format for the segmented progress bar with heading. This flow was a bit confusing when I tested with a screen reader and could be simplified to use the new standard and nest heading levels properly. Screenshots attached below.

---
**Step 1 of 2**

![Image](https://github.com/user-attachments/assets/91ae2211-b767-445a-8070-fbab164125ca)

---
**Step 2 of 2**

![Image](https://github.com/user-attachments/assets/f1c4e593-20cf-4ba7-a488-db7e37e1fe7d)

### Reproduction steps

1. Log into https://staging.va.gov/records/download-va-letters/letters/confirm-address with a staging user. I used 54.
2. Turn on the screen reader of your choosing
3. Listen to the headings in order:
    * If you're using VoiceOver, you can move by headings by pressing `Ctrl + Opt + Cmd + h`
    * If you're using NVDA or JAWS, you can move by pressing ` h `
4. Listen to the headings and verify the nesting is wrong, moving from H1 to H4 and back to H2. This can be confusing to screen reader users.

### Proposed solution or next steps

Refactor the headings to use the VADS [segmented progress bar with heading](https://design.va.gov/storybook/?path=/docs/uswds-va-segmented-progress-bar--default) styles and make sure the heading level is overridden to an H2. Remove the extra heading levels and test with screen readers for continuity.

### References, articles, or WCAG support

1. https://design.va.gov/storybook/?path=/docs/uswds-va-segmented-progress-bar--default
2. https://design.va.gov/storybook/?path=/docs/uswds-va-segmented-progress-bar--custom-header-level
3. Understanding [SC 1.3.1](https://w3.org/TR/WCAG22#info-and-relationships): Info and Relationships (Level A)

### Type of issue

- [ ] Axe-core
- [x] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [x] Content organization
- [ ] Focus
- [x] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [x] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by dysbo** [comment]
Looking for design review on this item as it seems pretty outdated

**Comment by sh0wn** [comment]
@dysbo @1Copenut This will be solved by moving towards the single step solution.
